### PR TITLE
feat: Add ma card link

### PR DESCRIPTION
**Pull request by sergigraciaj**
[Figma link](https://www.figma.com/file/7RhweKibaUnxX6WYBl7zKW/%E2%9A%A1%EF%B8%8F-Watt-Design-System?node-id=22008%3A9474&t=7aMGyZLjfGCVpwjA-4)

image

### CODE REVIEW

**Review by aquilinopintoa — APPROVED** [review]

**Comment by ferbg** [comment]
Good job! 🚀 
I would add the possibility to create a custom `CardLink` using a slot as suggested in _DS_

**Review by ferbg — APPROVED** [review]

**Review by alex-castelo — APPROVED** [review]
Top 🎉  Good job

**Comment by sergigraciaj** [comment]
> Good job! 🚀 I would add the possibility to create a custom `CardLink` using a slot as suggested in _DS_

@ferbg In Figma docs title is required, so what should we use as slot, all the content or just the picto and description?

File: src/components/MaCardLink/MaCardLink.vue [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<template>
+  <a class="ma-card-link" :href="href" :rel="computedRel">
+    <slot>
+      <div class="ma-card-link--content">
+        <div>
```
Comment (carlotamolintr) [inline src/components/MaCardLink/MaCardLink.vue:5]: Do we need this `<div>`?

Comment (sergigraciaj) [reply src/components/MaCardLink/MaCardLink.vue:5]: Yup, we use it to contain the picto/title/text in a column when using flex
image

**Review by carlotamolintr — APPROVED** [review]

**Pull request merged**